service/routes.py

```diff
@@ -17,17 +19,66 @@
 ######################################################################
 @app.route("/")
 def index():
-    """ Root URL response """
+    """ Promotion's Home URL response """
+    app.logger.info("Request for Promotion's Home Root URL")
     return (
-        "Reminder: return some useful information in json format about the service here",
-        status.HTTP_200_OK,
+        jsonify(
+            name="Promotion Rest API Service!!",
+            version="1.0"
+        ),
+        status.HTTP_200_OK
     )
-
+    
 
 ######################################################################
 #  R E S T   A P I   E N D P O I N T S
 ######################################################################
 
+# Place your REST API code here ...
+
+
+
+
+######################################################################
+# ADD A NEW Promotion
+######################################################################
+@app.route("/promotions", methods=["POST"])
+def create_promotion():
+    """
+    Creates a Promotion
+    This endpoint will create a Promotion based the data in the body that is posted
+    """
+    
+    app.logger.info("Request to create a promotion")
+    check_content_type("application/json")
+    
+    promo = Promotion()
+    promo.deserialize(request.get_json())
+    promo.create()
+    message = promo.serialize()
+    
+
+    app.logger.info("Promotion with ID [%s] created.", promo.id)
+    return jsonify(message), status.HTTP_201_CREATED
+
+
+def check_content_type(content_type):
+    """Checks that the media type is correct"""
+    if "Content-Type" not in request.headers:
+        app.logger.error("No Content-Type specified.")
+        abort(
+            status.HTTP_415_UNSUPPORTED_MEDIA_TYPE,
+            f"Content-Type must be {content_type}",
+        )
+
+    if request.headers["Content-Type"] == content_type:
+        return
+
+    app.logger.error("Invalid Content-Type: %s", request.headers["Content-Type"])
+    abort(
+        status.HTTP_415_UNSUPPORTED_MEDIA_TYPE,
+        f"Content-Type must be {content_type}",
```

Comment:
Note that the first case and the last case have the s ame results. There is a more concise way of doing this:

```
if no content type or content type != "...":
   abort 415
return
```
In fact, see how you don't need this helper in this way -- it's already a one liner and shows clear logic in the create function.